### ISSUE TITLE
Lumen compatibility

### ISSUE THREAD

**Issue by underdpt** [dmitry-ivanov/laravel-console-logger#6]
Hi,

I'm trying to test this package on Lumen 5.7 but it throws the exception: 

```
In MonologFormatter.php line 8:
 
  [ErrorException]
  Declaration of Illuminated\Console\Loggable\FileChannel\MonologFormatter::normalize($data) should be compatible with Monolog\Formatter\NormalizerF
  ormatter::normalize($data, $depth = 0)
 
 
Exception trace:
 Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}() at D:\desarrollos\bctpv\bcServer\vendor\illuminated\console-logger\src\Loggable\FileChan
nel\MonologFormatter.php:8
 include() at D:\desarrollos\bctpv\bcServer\vendor\composer\ClassLoader.php:444
 Composer\Autoload\includeFile() at D:\desarrollos\bctpv\bcServer\vendor\composer\ClassLoader.php:322
 Composer\Autoload\ClassLoader->loadClass() at n/a:n/a
```

Maybe Lumen is using a different Monolog version with a different `normalize()` declaration.

Also, this package extends `Illuminate\Foundation\Exceptions\Handler` that doesn't exists in Lumen. Lumen and Laravel both have a `app\Exceptions\Handler.php` that might be usable on `Illuminated\Console\Exceptions\ExceptionHandler.php` and would solve this issue.

Making these two changes on my test app did work (I've done a quick tests on Lumen only)

**Comment by dmitry-ivanov** [comment]
Hi!

I've fixed the first part, `5.7.5` released.

According to the `Illuminate\Foundation\Exceptions\Handler`... I'm not sure what solution can be used for the Lumen. Package can't rely on the `app\Exceptions\Handler.php`, because it's a project specifics, and the `App` namespace is just the default, which is customized by many applications.

Probably it was a mistake to rely on the `Illuminate\Foundation` classes, maybe I should use the contracts. I will take a look on that issue later.

Thanks!

**Comment by mstaack** [comment]
would love to see some improvements for lumen

**Comment by mstaack** [comment]
actually looking at the new log channel / stack implementation in lumen/laravel...pretty close

**Comment by alekstr** [comment]
Hey, is it possible to merge the normilize part fix to 5.2 branch? =) Thanks!

**Comment by dmitry-ivanov** [comment]
I think it's possible to use the decorator pattern for the ExceptionHandler, [for example, like here](https://github.com/nunomaduro/collision/blob/stable/src/Adapters/Laravel/CollisionServiceProvider.php#L44-L52).

In that case, the ExceptionHandler class might just implement the `Illuminate\Contracts\Debug\ExceptionHandler` contract, without extending the `Illuminate\Foundation\Exceptions\Handler` class.

Anyway, I prefer not to bother with Lumen for now because I don't use it at all.
